backend/src/main/java/gov/cdc/usds/simplereport/config/simplereport/DataHubConfig.java

```diff
@@ -0,0 +1,54 @@
+package gov.cdc.usds.simplereport.config.simplereport;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+import java.time.Instant;
+import java.util.Date;
+
+@ConfigurationProperties(prefix = "simple-report.data-hub")
+@ConstructorBinding
+public final class DataHubConfig {
+
+    private final boolean uploadenabled;
+    private final String uploadUrl;
+    private final int maxCsvRows;
+    private final String serviceUuid;
+    private final String apiKey;
+    private final String slackNotifyWebhookUrl;
+
+    public final Date EARLIEST_DATE = Date.from(Instant.parse("2020-01-01T00:00:00Z"));
```

Comment:
I think this should be passed in via the config and set to the timestamp from the last manual upload

Reply:
This is used as a default when the database is empty and for unit tests. I can change the name so it's usage is clear or I can add a comment.

Reply:
If we just need an arbitrarily early date, I think Date or DateTime probably has a constant for that? Might be out of range for pg though.